test/test_utils.py

```diff
@@ -2,7 +2,9 @@
 from dataclasses import dataclass, is_dataclass
 from enum import Enum
 
+import pytest
 from pydantic import BaseModel
+from unstructured.ingest.v2.interfaces import FileData
```

Comment:
Should we really import this data class from unstructured? Feels like we should rather copy and use our own implementation.